lib/game.rb

```diff
@@ -12,42 +12,94 @@ def initialize
   def welcome
     start = ""
     puts "Welcome to BATTLESHIP\nEnter p to play. Enter q to quit."
-    until start == "p" || start =="q" do
+    until start == "p" || start == "q" do
       print "> "
-      start = gets.chomp
+      start = gets.chomp.downcase
       puts "Please select valid option 'p' or 'q'."
```

Comment:
I like the updated downcase to gets.chomp to my suggested start.downcase.

Reply:
I'm glad you thought of that because this could have created a really annoying user interaction!